src/explorer/SubscriptionTreeItem.ts

```diff
@@ -61,7 +61,7 @@ export class SubscriptionTreeItem extends SubscriptionTreeItemBase {
         );
     }
 
-    public async createChildImpl(context: ICreateChildImplContext): Promise<AzureTreeItem> {
+    public async createChildImpl(context: ICreateChildImplContext & IDeployWizardContext): Promise<AzureTreeItem> {
```

Comment:
You should either change this to `Partial<IDeployWizardContext>` or use a separate `ICreateWebAppContext`.

I prefer a separate `ICreateWebAppContext` that has a an optional property `suppressCreateNotification`? I would trust that more than checking `context.fsPath` and that isolates `create` from `deploy` more cleanly.